src/domain-registry-essentials/architecture-overview.md

```diff
@@ -1,123 +1,135 @@
+Using Solidity smart contracts allows many new ways to manage subdomains. One of these ways is to use the `DomainZoneController` smart contract, which is supported by the Unstoppable Domains team.
 
-DomainZoneController suggests the model of ownership where subdomains can be managed only by their owners. Minting subdomains can be performed using whitelisted accounts, that can be added or renounced by smart contract admins.
+`DomainZoneController` implements a model of ownership where subdomains can be managed only by their owners. Minting subdomains can be performed using whitelisted accounts which can be added or removed by the smart contract administrators.
 
-Apart from minting subdomains, DomainZoneController provides functionality to set a Resolver address and manage records of a domain zone itself – these actions can be performed only by whitelisted accounts as well.
+In addition to minting subdomains, `DomainZoneController` provides functionality to set a `Resolver` address and manage the records of a domain zone itself. These actions can only be performed by whitelisted accounts.
 
 {% hint style="info" %}
-Internal addresses can't be accessed directly by a wallet, which implies that domains owned by smart contracts can only be managed with rules defined by smart contracts themselves. For example, one cannot transfer a subdomain if such functionality isn't defined by a smart contract's set of methods.
+Internal addresses can't be accessed directly by a wallet. Meaning domains owned by smart contracts can only be managed with rules defined by the smart contracts themselves. For example, one cannot transfer a subdomain if such functionality isn't defined by a smart contract's set of methods.
 {% endhint %}
 
-## Delegating domain management
+## Delegating Domain Management
+
+CNS allows users to delegate transaction execution to accounts that aren't domain owners.
 
-CNS allows users to delegate executing certain transactions to accounts that aren't domain owners. Registry and Resolver smart contracts implement methods that make use of [Meta transactions](../managing-domains/meta-transactions.md). One of the use-cases for using Meta transactions is delegating blockchain calls, that use gas, to other accounts. For domain owners, it means that they don't need to have funds on their accounts to manage their domains if another party takes responsibility to pay for executing transactions.
+`Registry` and `Resolver` smart contracts implement methods that make use of these [Meta Transactions](../managing-domains/meta-transactions.md). One use-case for meta transactions is delegating (gas-using) blockchain calls to other accounts. This allows domain owners to keep their domains and funds on separate accounts or even have someone else pay their transaction fees.
```

Comment:
> of these

Do we want to say "these"? To me, it doesn't sound like there's something restrictive about meta transactions

Reply:
`make use of` --> `use` 👍